web/rebuild_test.go

```diff
@@ -59,14 +64,64 @@ func TestSimulatedRebuildWorkPoolWithErrCount(t *testing.T) {
 	}
 }
 
+// prepareGitRepo creates copies src/repo folder to dst and initializes
+// dst/repo as a git repository and adds a single file lab1/lab1.go.
+func prepareGitRepo(src, dst, repo string) error {
```

Comment:
This function is identical to the one found in `fetcher_test.go`. Maybe these could be move to the fileop package?